src/lib/ngsi/Metadata.cpp

```diff
@@ -174,3 +174,46 @@ void Metadata::fill(const struct Metadata& md)
   value       = md.value;
   association = md.association;
 }
+
+
+
+/* ****************************************************************************
+*
+* renderV2 - 
+*/
+std::string Metadata::renderV2(ConnectionInfo* ciP, RequestType request, bool comma)
```

Comment:
ok, let's do the following:
I will try to reimplement this 'renderV2', using the method name toJson and with one single parameter (bool isLastElement)   (for the 'comma problem').